aten/src/ATen/native/README.md

```diff
@@ -277,6 +277,18 @@ them the same thing!)
 If two backends have the same dispatch function, you can write `CPU, CUDA: func`
 to reuse the same function name in both cases.
 
+TODO: expand this section.
+
+Available backend options can be found at
+https://github.com/pytorch/pytorch/blob/master/tools/codegen/gen.py#L970.
+In addition to backends above, we also support keyword `Math` which is an alias
+that maps to all backends. In other words, function registered to `Math` key
+should be a plain mathematical composition of other `at::` functions and works for any backend.
```

Comment:
(drive-by comment): Are things registered to the Math key required to be functional (e.g., no in-place operators), or are in-place operations OK?

Reply:
It doesn't have to be functional. I'd think of `Math` key being used to store the old `composite` implementations. 
Imagine we used to implement `native_group_norm` using `at::native_batch_norm` (it doesn't need to be functional), once we add an optimized kernel for certain backend, say
 ```
func: native_group_norm
CPU: group_norm_cpu
CUDA: group_norm_cuda
# Math: old_native_group_norm
```
`Math` is a way to tell all other backends you can still use `old_native_group_norm` without issues. Before this PR, once we add `CPU: group_norm_cpu`, backends like XLA are suddenly required to register a kernel to `native_group_norm` otherwise there's no kernel available for that key. 
Hope this explains it a little better, let me know if you have followup questions!